a.core_concepts.md

```diff
@@ -18,6 +18,7 @@ kubernetes.io > Documentation > Tasks > Access Applications in a Cluster > [Use
 
 ```bash
 kubectl create namespace mynamespace
+kubectl create ns mynamespace
```

Comment:
since it's the same command as the one in the previous line, can you add it with an explanatory comment?